relay-server/src/extractors/request_meta.rs

```diff
@@ -490,6 +503,18 @@ impl FromRequestParts<ServiceState> for PartialMeta {
 
         let ReceivedAt(received_at) = ReceivedAt::from_request_parts(parts, state).await?;
 
+        let signature =
+            parts
+                .headers
+                .get("x-sentry-relay-signature")
+                .map(|sig| match sig.to_str() {
+                    Ok(sig) => RelaySignature::Valid(RelaySignatureData::new(
+                        sig.to_owned(),
+                        bytes::Bytes::new(),
```

Comment:
The data here seems to be always empty, then why does it exist? Also that means `RelaySignatureData` would never work with a different signature (e.g. the signed body)?